package.json

```diff
@@ -28,15 +28,11 @@
   "license": "MIT",
   "dependencies": {
     "loader-utils": "^0.2.9",
-    "react-jade": "^2.4.0"
+    "react-jade": "2.x.x"
```

Comment:
Have you been having any difficulties with later versions of react-jade? Or did this help resolve the issues with npm3?